app/templates/ui/index.html

```diff
@@ -20,7 +20,7 @@
 
   </head>
   <body ng-app="app">
-
+    <div><toast></toast></div>
     <ui-view></ui-view>
 
     <!-- build:js js/lib.js -->
```

Comment:
I think the `<toast>` tag should be in root.html. Which means you will need to check the other themes as well..